library.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Arduino Abstractions",
+  "version": "0.0.1",
```

Comment:
Can we use semantic versioning here?

https://semver.org/spec/v2.0.0.html

I.e. the first version would be 0.1.0, the last digit indicates the "PATCH" number, so it should be zero on the first release.